src/error.rs

```diff
@@ -750,7 +754,7 @@ impl ToDiagnostic<FileId> for EvalError {
                 let mut msg = String::from("Blame error: ");
 
                 // Writing in a string should not raise an error, whence the fearless `unwrap()`
```

Comment:
(unrelated, but I saw the typo while going through the diff)
```suggestion
                // Writing in a string should not raise an error, hence the fearless `unwrap()`
```